# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/ipc/PWebrtcTCPSocket.ipdl
# Commit: f5353ea9f1df
# Full Hash: f5353ea9f1dfe15a65076bc7aa272d688ebdb8ca
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Add a proxy policy argument to NrSocketProxyConfig, and always set this config on the NrIceCtx. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45101
# ==============================================================================

diff -r ed1ae6a85255 -r f5353ea9f1df media/mtransport/ipc/PWebrtcTCPSocket.ipdl
--- a/media/mtransport/ipc/PWebrtcTCPSocket.ipdl	Tue Sep 17 18:17:13 2019 +0000
+++ b/media/mtransport/ipc/PWebrtcTCPSocket.ipdl	Tue Sep 17 18:17:14 2019 +0000
@@ -22,7 +22,8 @@
   async AsyncOpen(nsCString aHost,
                   int32_t aPort,
                   LoadInfoArgs aLoadInfoArgs,
-                  nsCString aAlpn);
+                  nsCString aAlpn,
+                  int proxyPolicy);
   async Write(uint8_t[] aWriteData);
   async Close();
 